ktor-client/ktor-client-features/ktor-client-json/ktor-client-serialization/api/ktor-client-serialization.api

```diff
@@ -9,6 +9,6 @@ public final class io/ktor/client/features/json/serializer/KotlinxSerializer : i
 }
 
 public final class io/ktor/client/features/json/serializer/KotlinxSerializer$Companion {
-	public final fun getDefaultJsonConfiguration ()Lkotlinx/serialization/json/JsonConfiguration;
```

Comment:
Can't we avoid breaking here? Does it actually makes sense (unsure)?

Reply:
Nope, this class will be completely removed in serialization with 1.4